### PR TITLE
T23: Voting contest

### PR DESCRIPTION
**Pull request by YoganathanM**

### CODE REVIEW

File: Assignment/Program.cs [hunk 1]
```diff
@@ -1,7 +1,21 @@
 ﻿namespace Assignment {
    internal class Program {
       static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+         Console.Write ("Enter total votes : ");
+         string input = Console.ReadLine ()!.ToLower ().Replace (" ", "");
```
Comment (rajeevramanTMM) [inline Assignment/Program.cs]: Fix this exception.
image

File: Assignment/Program.cs [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
    internal class Program {
       static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+         Console.Write ("Enter total votes : ");
+         string input = Console.ReadLine ()!.ToLower ().Replace (" ", "");
+         VotingContext (input);
+      }
+
+      public static void VotingContext (string input) {
+         Dictionary<char, int> vote = new ();
```
Comment (rajeevramanTMM) [inline Assignment/Program.cs]: Dictionary is not guaranteed that the order will be reserved. Re write using a list and tuple.

File: Assignment/Program.cs [hunk 3]
```diff
@@ -1,7 +1,21 @@
...
+         for (int i = 0; i < input.Length; i++) {
+            if (vote.ContainsKey (input[i]))
```
Comment (rajeevramanTMM) [inline Assignment/Program.cs]: if and else has only one statement you could write in the same line.

**Review by rajeevramanTMM — CHANGES_REQUESTED** [review]

File: Assignment/Program.cs (hunk 1 above)
Comment (YoganathanM) [reply Assignment/Program.cs]: Done

File: Assignment/Program.cs (hunk 2 above)
Comment (YoganathanM) [reply Assignment/Program.cs]: Re written using list

File: Assignment/Program.cs (hunk 3 above)
Comment (YoganathanM) [reply Assignment/Program.cs]: Done

**Review by rajeevramanTMM — APPROVED** [review]

**Pull request closed without merging**